package.json

```diff
@@ -16,7 +16,7 @@
     "gulp-babel": "^6.1.2",
     "gulp-concat": "^2.6.1",
     "gulp-uglify": "^3.0.0",
-    "materialize-css": "^0.99.0",
+    "materialize-css": "^1.0.0",
```

Comment:
`materialize-css` version must be > `1.0.0`

Reply:
`1.0.0` is the latest version